Lombiq.Tests.UI/Extensions/ElementRetrievalUITestContextExtensions.cs

```diff
@@ -101,6 +104,38 @@ public static void SucccessMessageExists(this UITestContext context, string matc
                 .Text
                 .ShouldBe(errorMessage);
 
+        /// <summary>
+        /// Retrieves the elements according to <paramref name="by"/> and matches their text content against <paramref
+        /// name="toMatch"/>. The strings are trimmed in both collections. If an item in <paramref name="toMatch"/> is
+        /// <see langword="null" /> it's ignored among the result elements too. Every other item is converted to string,
+        /// using invariant culture where possible.
+        /// </summary>
+        public static void VerifyElementTexts(this UITestContext context, By by, params object[] toMatch)
```

Comment:
Why is `toMatch` not `string[]`?

Reply:
In case you want to compare numbers too.

Reply:
I see, OK then.